datahub-web-react/src/app/entity/dataset/profile/schema/SchemaDescriptionField.tsx

```diff
@@ -37,11 +61,18 @@ export default function DescriptionField({ description, updatedDescription, onUp
     const [showAddModal, setShowAddModal] = useState(false);
 
     const onCloseModal = () => setShowAddModal(false);
+    const currentDesc: string = !!updatedDescription || updatedDescription === '' ? updatedDescription : description;
```

Comment:
how about just doing

```suggestion
    const currentDesc: string = !!updatedDescription ? updatedDescription : description;
```

since we aren't supporting saving blank descriptions anyway?